removalScript.js

```diff
@@ -5,7 +5,7 @@ if(window.location.href.split("//")[1].split("/")[0] == "www.youtube.com") {
 		let items = document.getElementById("items")
 		items.parentNode.removeChild(items)
 	} else if(window.location.href.split("?")[0].split("/")[3] == "") {
-		let yt = document.getElementById("primary")
-		yt.parentNode.removeChild(yt)
+		let yt = document.getElementsByTagName("ytd-two-column-browse-results-renderer")

```

Comment:
Ah whoops this needs to be fetch the index 0

Reply:
ok I'll update my code locally. That uh explains a lot actually.